src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,678 @@
+## Working with Files and Attachments:
+
+Many times you might want to create logical objects that have more complex data, such as images or videos, as part of their structure. For example, you might create a Person model with a profile picture, or a Post model that has an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file / attachment within the model to associate the record with the file.
```

Comment:
Let us remove the wording on complex here. I think that is confusing for customers, and makes it seem unattainable or difficult to implement.

Reply:
Just saw your comment, but I've already removed this. Thanks!